gradle.properties

```diff
@@ -1,6 +1,5 @@
 version=8.5.0
-iexecCommonVersion=8.5.0
-iexecCommonsPocoVersion=4.1.0
-
```

Comment:
I think you should keep the blank line, it appears to be a convention in all other files

Reply:
IntelliJ auto reformating ....

Reply:
Resolved in [9980248](https://github.com/iExecBlockchainComputing/tee-worker-post-compute/pull/121/commits/9980248fe156b58a36364c6468613b5c09fc82ad)